script.js

```diff
@@ -143,4 +144,24 @@ function getIds() { // this is the function to get the ids of every cocktail ava
     for (let i = 0; i < drinks.length; i++) {
         drinkIds.push(drinks[i].idDrink)
     }
-}
\ No newline at end of file
+}
+// Select multiple ingrediets from list
```

Comment:
This doesn't seem to work yet, if we are going to push this to the main branch can we commit it first so it doesn't cause errors in the page?